src/components/fields/RadioGroup.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import * as PropTypes from 'prop-types';
+import { RadioGroupContext } from '../FormContext';
+import Field from '../Field';
+
+class RadioGroup extends Component {
+    static propTypes = {
+        name: PropTypes.string.isRequired,
+        children: PropTypes.node.isRequired,
+        getValue: PropTypes.func.isRequired,
+        setValue: PropTypes.func.isRequired,
+        setTouched: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
+    };
+
+    static defaultProps = {
+        onChange: () => {},
+    };
+
+    chosenItem = value => {
+        const { setValue, setTouched, onChange } = this.props;
+        setValue(value);
+        setTouched(() => {
+            onChange(value);
+        });
+    };
+
```

Comment:
```
childApi = () => {
    const { name, getValue, setValue, setTouched, onChange } = this.props;

    return {
        name,
        getValue,
        onChange: event => {
          if (!event.target.checked) {
            return;
          }
          setValue(event.target.value, () => {
            if (onChange) {
              onChange(event);
            }
          });
        },
        onBlur: event => {
          setTouched(() => {
            if (onBlur) {
              onBlur(event);
            }
          });
        },
    };
};
```

Reply:
I tried to make component Radio more simple, only sent its value to RadioGroup. But method `setValue` accepts ability to receive data not from group but current selected radioItem. So radioItem becomes the same independent as it has been before.